src/conf_mode/service_ssh.py

```diff
@@ -95,6 +95,16 @@ def generate(ssh):
         syslog(LOG_INFO, 'SSH ed25519 host key not found, generating new key!')
         call(f'ssh-keygen -q -N "" -t ed25519 -f {key_ed25519}')
 
+    if 'trusted_user_ca_key' in ssh:
+        if ssh['trusted_user_ca_key'].startswith('http'):
+            result = call(f"curl --fail --silent -o /etc/ssh/trusted_user_ca_key {ssh['trusted_user_ca_key']}")
```

Comment:
Instead of hard coding a curl command, please use the `vyos.remote` module to download files.

Source for module here: https://github.com/vyos/vyos-1x/blob/current/python/vyos/remote.py

Reply:
I'll use this as a reference in another PR :)